src/handlers/localstorage/uniswap.js

```diff
@@ -22,6 +23,11 @@ export const getUniswapFavorites = network =>
 export const saveUniswapFavorites = favorites =>
   saveGlobal(UNISWAP_FAVORITES, favorites);
 
+export const getUniswapLists = network =>
+  getGlobal(UNISWAP_LISTS, DefaultTokenLists[network]);
+
+export const saveUniswapLists = lists => saveGlobal(UNISWAP_LISTS, lists);
```

Comment:
we don't need the changes in this file. `saveUniswapLists` is never used, as the DefaultTokenLists is set in userLists and we don't need the corresponding `lists` state in `redux/uniswap` (as it is taken care of in `redux/userLists`)